tests/python/unittest/test_runtime_rpc.py

```diff
@@ -191,9 +204,9 @@ def check(remote):
 
         remote.cpu().sync()
         with pytest.raises(AttributeError):
-            f3 = remote.system_lib()["notexist"]
+            func3 = remote.system_lib()["notexist"]  # pylint: disable=unused-variable
```

Comment:
can we just delete the func3 =?

Reply:
I didn't know if we wanted to change functionality of test, i.e. why is this here, would that throw an exception if something was broken? or is it just abandoned code?